contracts/Mars.sol

```diff
@@ -112,6 +112,19 @@ contract Mars is IbcReceiver, Ownable {
         dispatcher.closeIbcChannel(channelId);
     }
 
+    function initiateChannelHandshake(
+        IbcDispatcher dispatcher,
+        IbcReceiver portAddress,
+        string calldata version,
+        ChannelOrder ordering,
+        bool feeEnabled,
+        string[] calldata connectionHops,
+        CounterParty calldata counterparty,
+        Proof calldata proof
+    ) external {
+        dispatcher.openIbcChannel(portAddress, version, ordering, feeEnabled,connectionHops, counterparty, proof);
```

Comment:
@alfredo-stonk should I add a fee here as well?

Reply:
No fee for channel handshake while calling Dispatcher.  
We also removed packet fee processing from Dispatcher too, as it's handled by IBC middleware. Lots of refactoring to support general MW stack in #34. 

If dApps need a ics-29 fee-enabled channel, they'd call fee-MW's `openIbcChannel` func.

Let's deal no-fee only channel in this PR for now? We're prioritizing Catalyst's GeneralisedIncentive MW